tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/db/UnknownHeroException.java

```diff
@@ -21,4 +21,5 @@
     public UnknownHeroException(String message) {
         super(message);
     }
+
```

Comment:
Just whitespace - I'd remove this change from the commit to avoid noise.

Reply:
+1